Makefile

```diff
@@ -42,3 +42,52 @@ build:
 .PHONY: clean
 clean:
 	rm -rf integration/testdata/fixtures/
+
+$(GOBIN)/bbolt:
+	go get -v github.com/etcd-io/bbolt/cmd/bbolt
+
+DB_TYPE ?= trivy
+
+ifeq ($(DB_TYPE),trivy-light)
+	DB_ARG := --light
+else
+	undefine DB_ARG
```

Comment:
`undefine` directive is introduced in `make` 3.82. macOS installs `make` 3.81 as default. Is it possible not to use `undefine`?

Reply:
Good catch, probably that part isn't needed at all. I dropped it.